### ISSUE TITLE
Combining several languages in one example (grouping code)

### ISSUE THREAD

**Issue by artemlos** [EWSoftware/SHFB#85]
I am trying to construct an example demonstrated in different programming languages. 
Is there a way to display each code snippet as a tab item, as shown below:
![lang_ex_github](https://cloud.githubusercontent.com/assets/6392760/7444424/be9bd31e-f181-11e4-9734-1712039100db.PNG)

Right now, I am using the following code. Unfortunately, each code snippet is displayed on its own, not as a a tab item. Is there any way to group code snippet version into a "tab-control" style?  
````
        /// <example>
        /// Message
        /// <code language="VB.NET">
        /// Example in VB.NET
        /// </code>
        /// <code langauge="cs" title="C#">
        /// Example in C# .NET
        /// </code>
        /// </example>
````

**Comment by EWSoftware** [comment]
If you want it grouped, you must omit the title attribute on the C# example.  Since in this case the title is the language, you'll get the same title since the default is the language name.

**Comment by artemlos** [comment]
I've tried to remove the title from the C# code but the code snippets are still split up. Is there a certain tag I should use to group them? Here's how it looks now:
![lang_ex_github2](https://cloud.githubusercontent.com/assets/6392760/7446504/011bce7a-f1db-11e4-92e6-974af3ff2bc6.PNG)

**Comment by EWSoftware** [comment]
It looks like you left the title blank.  When I said remove the title attribute, I meant completely as in the VB code example.  The only attribute should be the language.

**Comment by artemlos** [comment]
Here's the entire code:

```
        /// <example>
        /// Machine code can be calculated with the function below. Any other hash algorithm will do, as long as it only contains letters and digits only.
        /// 
        /// <code language="VB.NET">
        /// 'eg. "61843235" (getEightDigitsLongHash)
        /// 'eg. "D38F13CAB8938AC3C393BC111E1A85BB4BA2CCC9" (getSHA1)
        /// Dim machineCode = SKGL.SKM.getMachineCode(AddressOf SKGL.SKM.getEightDigitsLongHash)
        /// Dim machineCode = SKGL.SKM.getMachineCode(AddressOf SKGL.SKM.getSHA1)
        /// </code>
        /// <code langauge="cs">
        /// //eg. "61843235" (getEightDigitsLongHash)
        /// //eg. "D38F13CAB8938AC3C393BC111E1A85BB4BA2CCC9" (getSHA1)
        /// string machineID1 = SKGL.SKM.getMachineCode(SKGL.SKM.getEightDigitsLongHash);
        /// string machineID2 = SKGL.SKM.getMachineCode(SKGL.SKM.getSHA1);
        /// </code>
        /// </example>
```

**Comment by EWSoftware** [comment]
I didn't see it in the original.  The "language" attribute is misspelled on the C# example.

**Comment by artemlos** [comment]
Oh, I did not see it! I've fixed the spelling and now it looks as following:
![lang_ex_github3](https://cloud.githubusercontent.com/assets/6392760/7446808/ae999ad8-f1e4-11e4-970d-7ee928ac22cd.PNG)

**Comment by EWSoftware** [comment]
What presentation style and what version of the help file builder are you using?  Using VS2013 in the latest release, I get the expected results (tabbed example with C# and VB.NET).

**Comment by artemlos** [comment]
I use VS2013 theme. I think I use an older release (last one on https://github.com/EWSoftware/SHFB/releases). I will try to upgrade and let you know if it fixed the issue.

**Comment by EWSoftware** [comment]
No further response so I'm closing this for now.  Re-open if it's still an issue.

**Comment by StevenBonePgh** [comment]
Eric - I saw similar behavior where languages were stacked instead of tabbed, and it is due to a setting problem.  For each language you wish to group, the `Syntax Filter` on the `Help File` tab of the SHFB project properties must be checked.  The way I figured that out was to build the 'Sandcastle MAML Guide' and look at the `Block Elements` => `codeReference` topic which renders properly.  Then I compared the project settings and noticed it was not checked.  I don't know if you consider this is a bug or not, but certainly a notice in the `codeReference `topic and the `code `topic (which does not have an example of grouping as you described it above) would be helpful for future folks.
